### PR TITLE
Various improvements to the LLVM 3.4 install page.  Dropped the mention of metaSMT being experimental.

### PR DESCRIPTION
**Pull request by ccadar**

### CODE REVIEW

File: build-llvm34.md [hunk 1]
```diff
@@ -54,9 +55,10 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
 
    KLEE supports multiple different constraint solvers. You must install at least one to build KLEE.
 
-   * [STP](https://github.com/stp/stp) Historically KLEE was built around only this solver so support for this solver is the most stable. For build instructions see [here]({{site.baseurl}}/build-stp).
-   * [Z3](https://github.com/z3prover/z3) Z3 support is much more recent addition to KLEE but is reasonably stable. You should use Z3 version >= 4.4. For build instructions see [here](https://github.com/Z3Prover/z3/blob/master/README.md).
-   * [metaSMT](https://github.com/agra-uni-bremen/metaSMT) **experimental** For build instructions see [here](https://github.com/agra-uni-bremen/metaSMT).
+   * [STP](https://github.com/stp/stp) Historically KLEE was built around STP so support for this solver is the most stable. For build instructions see [here]({{site.baseurl}}/build-stp).
+   * [Z3](https://github.com/z3prover/z3) is a more recent addition to KLEE but is reasonably stable. You should use Z3 version &ge; 4.4. For build instructions see [here](https://github.com/Z3Prover/z3/blob/master/README.md).
```
Comment (delcypher) [inline build-llvm34.md]: Why did you change `>=` with `&ge`? That's unnecessary. This is markdown (processed by Kramdown) so we don't need to use HTML unless absolutely necessary.

File: build-llvm34.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
 
    **NOTE:** KLEE is currently tested on **Linux x86-64**, and might break on x86-32.
 
-   If you are using a recent Ubuntu (≥ 12.04 and ≤ 15.10, e.g. 14.04 LTS) or Debian, we recommend you to use the LLVM packages provided by LLVM itself. Use [LLVM Package Repository](http://llvm.org/apt/) to add the appropriate line to your `/etc/apt/sources.list`. As an example, for Ubuntu 14.04, the following lines should be added:  
+   If you are using a recent Ubuntu (≥ 12.04 and ≤ 15.10, e.g. 14.04
+   LTS), Debian or MacOS/Darwin, we recommend you to use the LLVM packages provided by LLVM itself. Use [LLVM Package Repository](http://llvm.org/apt/) to add the appropriate line to your `/etc/apt/sources.list`. As an example, for Ubuntu 14.04, the following lines should be added:
```
Comment (delcypher) [inline build-llvm34.md]: This does not make sense. There are know APT packages for MacOS/Darwin.

Comment (ccadar) [inline build-llvm34.md]: Good catch, there was no mention of MacOS, but that's the wrong place to add it.

File: build-llvm34.md (hunk 1 above)
Comment (ccadar) [inline build-llvm34.md]: This shows as two chars in my browser.  We can replace it with the UNICODE char, if you prefer.

**Comment by ccadar** [comment]
Should be ready to merge

File: build-llvm34.md (hunk 1 above)
Comment (delcypher) [inline build-llvm34.md]: Something is wrong with your setup. If you look at http://klee.github.io/build-llvm34/ the `>=` displays fine.

Comment (ccadar) [inline build-llvm34.md]: That's where I looked.  It shows as two chars, both in Chrome and Firefox.  In any case, I changed it to the unicode char.

File: build-llvm34.md [hunk 3]
```diff
@@ -54,9 +54,10 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
 
    KLEE supports multiple different constraint solvers. You must install at least one to build KLEE.
 
-   * [STP](https://github.com/stp/stp) Historically KLEE was built around only this solver so support for this solver is the most stable. For build instructions see [here]({{site.baseurl}}/build-stp).
-   * [Z3](https://github.com/z3prover/z3) Z3 support is much more recent addition to KLEE but is reasonably stable. You should use Z3 version >= 4.4. For build instructions see [here](https://github.com/Z3Prover/z3/blob/master/README.md).
-   * [metaSMT](https://github.com/agra-uni-bremen/metaSMT) **experimental** For build instructions see [here](https://github.com/agra-uni-bremen/metaSMT).
+   * [STP](https://github.com/stp/stp) Historically KLEE was built around STP so support for this solver is the most stable. For build instructions see [here]({{site.baseurl}}/build-stp).
+   * [Z3](https://github.com/z3prover/z3) is a more recent addition to KLEE but is reasonably stable. You should use Z3 version ≥ 4.4. For build instructions see [here](https://github.com/Z3Prover/z3/blob/master/README.md).
+   * [metaSMT](https://github.com/agra-uni-bremen/metaSMT) supports
```
Comment (delcypher) [inline build-llvm34.md]: You probably ought to note the version of metaSMT required.

Comment (ccadar) [inline build-llvm34.md]: Good idea, I added the version used on TravisCI.

**Comment by ccadar** [comment]
Should we merge this, @delcypher ?

**Comment by delcypher** [comment]
@ccadar LGTM. Merging.
